storage/api/csk_operations.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
```

Comment:
I can't tell with github, but please make sure you're using 4 spaces per tab. You can run `tox -e pep8` to test.